inausoft.netCLI.all/inausoft.netCLI/MateuszCommands/MateuszCommandHandler.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+
+namespace inausoft.netCLI.MateuszCommands
+{
+    public class MateuszCommandHandler : IMateuszCommandHandler<MateuszCommandHandler.MateuszCommand>
+    {
+
+        private MateuszCommand command;
+
+        public MateuszCommandHandler()
+        {
+            command = new MateuszCommand();
```

Comment:
This is a original sin of the proposed solution. I think it is important to separate `IHandler` from initializing command.

Reply:
I think that the shouldn't be separated - i event think that there is no benefit of having separate `Command` class, so I even consider `CommandHandlers` to contains `Command` properties

Reply:
Handler should be able to handle multiple commands in it's lifespan - hence having command as property is not best idea.

Reply:
CLI application will run, execute command and then close. I don't think that any handler will handle multiple commands then.